certified-connectors/Strategic Portfolio Manager/apiDefinition.swagger.json

```diff
@@ -1,13 +1,13 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v2.0",
-    "title": "Strategic Portfolio Manager",
-    "description": "Strategic Portfolio Manager enables you to continuously connect, align and orchestrate all investments with strategies to drive business agility more effectively. It connects disparate silos across the enterprise and provides 360 degrees of insight to help ensure that all investments are derived from \u2013 and continuously align with \u2013 strategic priorities. Connect to Strategic Portfolio Manager to plan, manage, and adjust your initiatives, projects, epics and more.",
+    "version": "v3.0",
+    "title": "NH360 Strategic Portfolio Manager",
```

Comment:
The 'title' value goes over the character limit '30', please refer this link and correct the same. 

The 'title' value goes over the character limit '30' ValuePath: /info/title